server/seeds/demoSeed.js

```diff
@@ -0,0 +1,26 @@
+const dotenv = require('dotenv').config({path:__dirname+'/./../.env'})
+const mongoose = require("mongoose");
+const User = require("../models/User");
+const bcrypt = require("bcryptjs");
+
+
+const seedDB = async (req, res, next) => {
+  const salt = await bcrypt.genSalt(10);
+  saltedPassword = await bcrypt.hash(process.env.DEMO_USER_PASSWORD, salt);
+  
+  const demoUser = [
+    {
+      email: process.env.DEMO_USER_EMAIL,
+      password: saltedPassword,
+      name: "demo",
+    },
+  ]
+
+  const conn = await mongoose.connect(process.env.MONGO_URI)
+  console.log(`MongoDB Connected: ${conn.connection.host}`);
+  await User.insertMany(demoUser);
```

Comment:
We should also be creating a demo profile record in the db as well. This can just have pre-filled values such as the name being like John Doe.

Reply:
got it - updated demoSeed.js so that it creates a corresponding demo profile record after we save the demo user.